t/t5403-post-checkout-hook.sh

```diff
@@ -74,6 +79,25 @@ test_expect_success 'post-checkout receives the right args when not switching br
 	test $old = $new && test $flag = 0
 '
 
+test_expect_success 'post-checkout is triggered on rebase' '
+	GIT_DIR=clone2/.git git checkout -b rebase-test master &&
```

Comment:
Please use `git -C clone2 [...]` instead.

Reply:
Removed the clones. They're not really needed...